pkgs/pkgs-lib/formats/libconfig/test/comprehensive/default.nix

```diff
@@ -63,12 +63,16 @@ in
 
     doCheck = true;
     checkPhase = ''
-      diff -U3 ${./expected.txt} ${libconfig-test-cfg}
+      cp ${./expected.txt} expected.txt
+      substituteInPlace expected.txt \
+          --replace "Placeholder to be replaced with the path of the libconfig-test-include file" \
```

Comment:
The convention for replacements like these is usually `@whatever@` and then `--subst-var whatever`.

Reply:
done, that does look better (: